jerry-math/jerry-math-internal.h

```diff
@@ -74,7 +76,13 @@ typedef union
 #endif /* __LITTLE_ENDIAN */
 
 #ifndef NAN
-#define NAN (0.0/0.0)
+#ifdef _MSC_VER
+#define INFINITY    ((float) (1e+300 * 1e+300)) /* 1e+300*1e+300 must overflow */
+#define NAN         ((float) (INFINITY * 0.0f))
+#else
```

Comment:
(minor) If the PR gets updated, it would be nice to add a comment here: `/* !_MSC_VER */`

Reply:
this suggestion did not get applied